team-build/src/components/Form.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState, ChangeEvent, ReactNode } from "react";
+import { string } from "prop-types";
```

Comment:
what is prop-types?

Reply:
it was auto-imported from one of my extensions, should've removed since it is unused. prop-types is a package that allows you to warn users of your app of certain props and their types, but TypeScript really just enforces it better

Reply:
You have to do something like 
```js
Form.propTypes = {
...
}
```
docs: https://reactjs.org/docs/typechecking-with-proptypes.html